<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="151" started-at="2021-10-28T06:04:34Z" finished-at="2021-10-28T06:04:34Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="151" started-at="2021-10-28T06:04:34Z" finished-at="2021-10-28T06:04:34Z">
      <class name="com.azure.AppTest">
        <test-method status="FAIL" signature="azurTest()[pri:0, instance:com.azure.AppTest@419c5f1a]" name="azurTest" duration-ms="115" started-at="2021-10-28T06:04:34Z" finished-at="2021-10-28T06:04:34Z">
          <exception class="java.lang.IllegalStateException">
            <message>
              <![CDATA[topicName (dev-eps-azure-sb-pushapi-notification-topic) must have a subscriptionName associated with it.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalStateException: topicName (dev-eps-azure-sb-pushapi-notification-topic) must have a subscriptionName associated with it.
at com.azure.messaging.servicebus.ServiceBusClientBuilder.getEntityPath(ServiceBusClientBuilder.java:516)
at com.azure.messaging.servicebus.ServiceBusClientBuilder.access$1500(ServiceBusClientBuilder.java:85)
at com.azure.messaging.servicebus.ServiceBusClientBuilder$ServiceBusReceiverClientBuilder.buildAsyncClient(ServiceBusClientBuilder.java:1418)
at com.azure.messaging.servicebus.ServiceBusClientBuilder$ServiceBusReceiverClientBuilder.buildAsyncClient(ServiceBusClientBuilder.java:1394)
at com.azure.messaging.servicebus.ServiceBusProcessorClient.<init>(ServiceBusProcessorClient.java:107)
at com.azure.messaging.servicebus.ServiceBusClientBuilder$ServiceBusProcessorClientBuilder.buildProcessorClient(ServiceBusClientBuilder.java:1241)
at com.rmg.TestManager.BusManager.receiveMessages(BusManager.java:110)
at com.azure.AppTest.azurTest(AppTest.java:25)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalStateException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- azurTest -->
      </class> <!-- com.azure.AppTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
